### ISSUE TITLE
sar tool doesn't work in pbench-agent-0.40

### ISSUE THREAD

**Issue by jeremyeder** [distributed-system-analysis/pbench#401]
```
[error][20161005_12:49:49.431069750] [check_install_rpm] the installation of pbench-sysstat-11.2.0 failed
For some reason this tool could not be installed
[error][20161005_12:49:50.254381670] [check_install_rpm] the installation of pbench-sysstat-11.2.0 failed
For some reason this tool could not be installed
[error][20161005_12:49:51.071011676] [check_install_rpm] the installation of pbench-sysstat-11.2.0 failed
For some reason this tool could not be installed
[error][20161005_12:49:51.885403641] [check_install_rpm] the installation of pbench-sysstat-11.2.0 failed
For some reason this tool could not be installed
proc-vmstat tool is now registered in group default
proc-interrupts tool is now registered in group default
turbostat tool is now registered in group default
perf tool is now registered in group default
```

It's because pbench-sysstat version mismatch in this file:
```
/opt/pbench-agent/tool-scripts/sar
```

Change that to ```tool_package_ver="11.5.1"``` and it seems to work after that.

```
# rpm -qa|grep pbench
pbench-fio-2.14-9.x86_64
pbench-sysstat-11.5.1-1.x86_64
pbench-agent-internal-0.40-1.noarch
pbench-agent-0.40-1gf281562.noarch
```

**Comment by ndokos** [comment]
Not sure why that is: I did build an 11.5.1 RPM last week, but there is an 11.2.0 RPM as well, so it should pick that one up. I've opened PR #403 to address this, but I'd like to find out what's going wrong so I'm not planning to merge it immediately. I'll update this issue as soon as I find out what's going on.

**Comment by ndokos** [comment]
The pbench agent RPM requires pbench-sysstat (unspecified version), so it installed the latest pbench-sysstat (11.5.1 after I built it last Friday). Then sar wants 11.2.0 and fails. I'll get rid of the require in the spec file and let the sar tool script make the final decision.

The only remaining question is whether to commit the version bump in the sar tool script: I could delete the 11.5.1 build from COPR and leave it at 11.2.0 (the people who have already installed it will still get the  error though).  (later) Nope: I tried it and it cannot find the 11.2.0 version - why?

I'll merge PR #403 and build a new pbench-agent RPM.
